src/components/scenes/CoinRankingDetailsScene.tsx

```diff
@@ -235,7 +235,6 @@ const CoinRankingDetailsSceneComponent = (props: Props) => {
     stakingWallets.some(
```

Comment:
What if we change `some` to `every`. As soon as even a single wallet has a policy, we call it "good enough" and show a rate. In that case, we might not even need to remove `isLoading`.

I see how removing `isLoading` speeds things up, but it doesn't feel the best. Still, I approve because it does get the job done.

Reply:
Your suggestion seems to work